pynucastro/networks/rate_collection.py

```diff
@@ -639,6 +644,10 @@ def _build_collection(self):
                 if nuc not in self.unique_nuclei:
                     self.unique_nuclei.append(nuc)
 
+        if self.reliable_spins:
+            for n in self.unique_nuclei + self.approx_nuclei:
```

Comment:
I don't _think_ we have to include self.approx_nuclei in here, because I don't think approximated-out nuclei are used in either derived rates or NSE, so I don't think there's occasion to reference the spin of approx_nuclei. Nevertheless, I'm leaving it in, because it'd be a pretty obscure bug if we did want to check the spin for some reason and didn't realize the reliable_spin flag was wrong.